docs/docs/Configuration/environment-variables.md

```diff
@@ -179,7 +179,7 @@ The following table lists the environment variables supported by Langflow.
 | Variable | Format | Default | Description |
 |----------|--------|---------|-------------|
 | <Link id="DO_NOT_TRACK"/>DO_NOT_TRACK | Boolean | `false` | If this option is enabled, Langflow does not track telemetry. |
-| <Link id="LANGFLOW_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>AUTO_LOGIN | Boolean | `true` | Enable automatic login for Langflow. Set to `false` to disable automatic login and require the login form to log into the Langflow UI. Setting to `false` requires [`LANGFLOW_SUPERUSER`](#LANGFLOW_SUPERUSER) and [`LANGFLOW_SUPERUSER_PASSWORD`](environment-variables.md#LANGFLOW_SUPERUSER_PASSWORD) to be set. |
+| <Link id="LANGFLOW_AUTO_LOGIN"/><span class="env-prefix">LANGFLOW_</span>AUTO_LOGIN | Boolean | `true` | Enable automatic login for Langflow. Set to `false` to disable automatic login and require the login form to log into the Langflow UI. Setting to `false` requires [`LANGFLOW_SUPERUSER`](#LANGFLOW_SUPERUSER) and [`LANGFLOW_SUPERUSER_PASSWORD`](environment-variables.md#LANGFLOW_SUPERUSER_PASSWORD) to be set. Langflow API requests require API keys as of Langflow v1.5. |
```

Comment:
This is on multiple pages. it might be best to have the full explanation in one place (maybe here https://github.com/langflow-ai/langflow/pull/8721/files#diff-bed09f7f66b7dd38c04ae88b5ed5374139a4765896e188c310154d9074b92211R25 ) and then link all other mentions of it to that SSOT.